.github/workflows/update-beta-packages.yml

```diff
@@ -0,0 +1,42 @@
+name: Update npm Packages
+
+on:
+  repository_dispatch:
+    types: [update-packages-beta]
+
+jobs:
+  update-packages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install yarn
+        run: sudo npm install -g yarn
+      
+      - name: Install cargo
+        uses: baptiste0928/cargo-install@v2
+        with:
+          crate: cargo
+
+      - name: Add npm package
+        run: yarn add @polybase/polylang@beta
```

Comment:
auth does not use polylang, have a look at the package.json, for this one its:

  "@polybase/client"
  "@polybase/eth"
  "@polybase/react"
  "@polybase/util"